test/test_rtfcre_dict.py

```diff
@@ -5,24 +5,67 @@
 
 import pytest
 
-from plover.dictionary.rtfcre_dict import RtfDictionary, format_translation
+from plover.dictionary.rtfcre_dict import RtfDictionary, TranslationFormatter
+from plover.dictionary.rtfcre_parse import BadRtfError
 
 from plover_build_utils.testing import dictionary_test, parametrize
 
 
 @parametrize((
     lambda: ('', ''),
-    lambda: ('{^in^}', r'\cxds in\cxds '),
-    lambda: ('{pre^}', r'pre\cxds '),
-    lambda: ('{pre^} ', r'pre\cxds '),
-    lambda: ('{pre^}  ', r'pre\cxds '),
+    # Affix handling.
+    lambda: ('{^}', r'{\cxds}'),
+    lambda: ('{^^}', r'{\cxds}'),
+    lambda: ('{^in^}', r'{\cxds in\cxds}'),
+    lambda: ('{pre^}', r'{pre\cxds}'),
+    lambda: ('{pre^} ', r'{pre\cxds}'),
+    lambda: ('{pre^}  ', r'{pre\cxds}'),
+    lambda: ('{ pre ^}  ', r'{ pre \cxds}'),
+    lambda: ('{^post}', r'{\cxds post}'),
+    lambda: (' {^post}', r'{\cxds post}'),
+    lambda: ('  {^post}', r'{\cxds post}'),
+    lambda: ('{^ post }  ', r'{\cxds  post }'),
+    # Escaping special characters.
+    lambda: (r'\{', r'\\\{'),
+    lambda: (r'\}', r'\\\}'),
+    # Handling newlines.
+    lambda: ('test\nsomething', r'test\line something'),
+    lambda: ('test\n\nsomething', r'test\par something'),
+    lambda: ('test\\nsomething', r'test\\nsomething'),
+    # Handling tabs.
+    lambda: ('test\tsomething', r'test\tab something'),
+    lambda: ('test\\tsomething', r'test\\tsomething'),
+    # Infix with force cap.
+    lambda: ('{^\n^}{-|}', r'{\cxds \line \cxds}\cxfc '),
+    lambda: ('{^\n\n^}{-|}', r'{\cxds \par \cxds}\cxfc '),
+    # Plover custom formatting:
+    # - meta: command
+    lambda: ('{PLOVER:TOGGLE}', r'{\*\cxplovermeta PLOVER:TOGGLE}'),
```

Comment:
`\cxplovercommand` maybe? and the `PLOVER:` is redundant here.

Reply:
No, the goal is to avoid the need for a specific control word for each feature supported by metas, so only one general `\cxplovermeta` control word, with the content.